pyproject.toml

```diff
@@ -10,6 +7,18 @@ target-version = ['py312']
 # PIP:
 #   pip install -e .[dev]
 
+################################################################################
+# the following sections do not contain unique configurations to this package.
+################################################################################
```

Comment:
Two things:
1. Can the `#`s line up with the text? Or maybe have a terminating `#` to make it look like a frame? Haha.
2. Should we mention that this section is the boilerplate `pyproject.toml` for all the ScreenPy extensions? It feels weird saying "this is not unique" without explaining why we're saying that.

Reply:
Something along the lines of this?
```
################################################################################
# START boilerplate
################################################################################
...
################################################################################
# END boilerplate
################################################################################
```


Overall my one concern -- and I don't have a good solution for how to handle it -- would be if any of the projects need specific configuration within the boilerplate sections.  As of now, screenpy has one extra variable in `[tool.ruff.lint.per-file-ignores]` from all the other projects.  

I'm not sure what to do if more occur (in any of the screenpy packages). My thinking is we cross that bridge if/when it happens, but we'll likely need to include big bold comments around the parts that don't match the boilerplate.

Reply:
I just realized what you were asking about lining up with text.  Do you mean this:
```
#############################################
# START OF BOILERPLATE ScreenPyHQ CONFIGURATIONS.
#############################################
```
I picked 88 chars so it's VERY noticeable.   Meaning, if you edit something in the boilerplate, we had better copy it to all the other packages.  I was hoping it would also make it a little easier to copy/paste the whole boilerplate.